.cirrus.yml

```diff
@@ -618,12 +618,6 @@ compose_test_task:
     depends_on: *build
     gce_instance: *standardvm
     matrix:
-      - env:
-            TEST_FLAVOR: compose
-            PRIV_NAME: root
-      - env:
-            TEST_FLAVOR: compose
-            PRIV_NAME: rootless
       - env:
             TEST_FLAVOR: compose_v2
```

Comment:
`TEST_FLAVOR` could now be deduplicated, down to the task `env` block instead of matrix, but this is minor

Reply:
fixed and the typo is also fixed